langfuse/callback/langchain.py

```diff
@@ -991,14 +991,8 @@ def _log_debug_event(
         parent_run_id: Optional[UUID] = None,
         **kwargs,
     ):
-        kwargs_log = (
-            ", " + ", ".join([f"{key}: {value}" for key, value in kwargs.items()])
-            if len(kwargs) > 0
-            else ""
-        )
         self.log.debug(
             f"Event: {event_name}, run_id: {str(run_id)[:5]}, parent_run_id: {str(parent_run_id)[:5]}"
-            + kwargs_log
```

Comment:
Why did you remove this?

Reply:
expensive calculation, added overhead in multiple hundreds of milliseconds for large objects